admin/convert_uoffset_to_crystal_cartesian.m

```diff
@@ -1,6 +1,6 @@
 function convert_uoffset_to_crystal_cartesian(filename,varargin)
 % Read uoffset, currently stored in sqw files, and convert this offset into
-% crystal cartesian system coordinat if the value, specified in the sqw
+% crystal Cartesian system coordinate if the value, specified in the sqw
```

Comment:
```suggestion
% crystal Cartesian system coordinates if the value, specified in the sqw
```
Following this change it says "if..." but it the sentence is incomplete and does not say what this condition is